Library/Formula/pkcrack.rb

```diff
@@ -0,0 +1,99 @@
+class Pkcrack < Formula
+  homepage "https://www.unix-ag.uni-kl.de/~conrad/krypto/pkcrack.html"
+  url "https://www.unix-ag.uni-kl.de/~conrad/krypto/pkcrack/pkcrack-1.2.2.tar.gz"
+  sha256 "4d2dc193ffa4342ac2ed3a6311fdf770ae6a0771226b3ef453dca8d03e43895a"
+
+  # Apply to the patch to build on the OSX
+  # I rewrite "#include<malloc.h>" to "#include<stdlib.h>" because OSX doesn't have 'malloc.h'
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!